src/NodeVisitors/Attributes.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\twig_fractal\TokenParser\Render.
+ */
+
+namespace Drupal\twig_fractal\NodeVisitors;
+
+use Twig_BaseNodeVisitor;
+use Twig_Environment;
+use Twig_Node;
+use Twig_Node_Expression_Filter;
+
+class Attributes extends Twig_BaseNodeVisitor {
+
+  protected $isAttribute = TRUE;
+
+  /**
+   * Checks if a node contains an attributes name expression.
+   *
+   * @return Twig_Node The modified node
+   */
+  public function doEnterNode(Twig_Node $node, Twig_Environment $env): object {
+    if ($node->hasAttribute('name') && $node->getAttribute('name') === 'attributes') {
```

Comment:
Should this also cover all variables having "attributes" as suffix in their name as they are converted as well; e.g., `link_attributes`?